media-libs/libplacebo/libplacebo-0.6.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+SCM=""
+if [[ "${PV}" == "9999" ]]; then
+	SCM="git-r3"
```

Comment:
inherit it from there

Reply:
That's the way I had it originally, but I was given this feedback by user @kagami (for a different but very similar ebuild, media-libs/dav1d-9999):

```
2018-09-24 08:54:07	kagami	haasn: inherit git-r3 should be first line btw
2018-09-24 08:54:17	kagami	see man for inherit
2018-09-24 08:57:36	kagami	or see how it's done in ffmpeg ebuild (SCM variable)
...
2018-09-27 15:42:14	kagami	haasn: https://0x0.st/s3kO.ebuild
```

I can change it back to inheriting from the `if` if you still prefer it that way.